<launch>
  <node name="okvis_node" pkg="msckf" type="okvis_node" output="screen">
    <param name="config_filename" value="$(find msckf)/config/config_fpga_p2_euroc_copy.yaml" />
    <param name="mesh_file" value="firefly.dae" />
    <remap from="/camera0" to="/cam0/image_raw" />
    <remap from="/camera1" to="/cam1/image_raw" />
    <remap from="/calibration0" to="/cam0/calibration" />
    <remap from="/calibration1" to="/cam1/calibration" />
    <remap from="/imu" to="/imu0" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find msckf)/config/rviz.rviz" output="log">
  </node>

</launch>
